turbine/src/cluster_nodes.rs

```diff
@@ -716,9 +725,123 @@ mod tests {
     use {
         super::*,
         itertools::Itertools,
-        std::{fmt::Debug, hash::Hash},
+        std::{collections::HashSet, fmt::Debug, hash::Hash},
         test_case::test_case,
     };
+    #[test]
+    fn test_dedup_tvu_addrs_across_protocols() {
```

Comment:
Which of these tests still pass with the old code?

Reply:
test_dedup_tvu_addrs should pass, others will fail.